transport-native-unix-common/src/main/c/netty_unix_socket.c

```diff
@@ -321,14 +336,6 @@ int netty_unix_socket_getOption(JNIEnv* env, jint fd, int level, int optname, vo
     return rc;
 }
 
-int netty_unix_socket_setOption(JNIEnv* env, jint fd, int level, int optname, const void* optval, socklen_t len) {
```

Comment:
Sure, it doesn't have to, but some teams prefer to keep related code adjacent / contiguous if possible.  If you don't have such a requirement I can move the bare minimum.